tweets/2019/03/tweet-with-us.tweet

```diff
@@ -0,0 +1,2 @@
+Have something interesting for nock community? Submit a tweet on GitHub:
+https://github.com/nock/twitter/tree/master/tweets#readme
```

Comment:
so maybe we should just link to the repository and improve its README there? 

We have a private WIP twitter repo for Probot, the READMe looks like this

![image](https://user-images.githubusercontent.com/39992/54079589-37c8a480-4294-11e9-951d-a527598a4197.png)

Reply:
That would make sense for me. But we can have multiple tweets. It's not a problem to do so, right?

Can twitterbot make threads?

Reply:
Not yet, but something I’d love to add support for in future